stack.yaml

```diff
@@ -0,0 +1,3 @@
+resolver: lts-13.9
```

Comment:
this seems silly; why doesnt something like `stack install --resolver lts-13.9` "just work" without needing a stack.yaml boilerplate?

Reply:
By having a `stack.yaml` in the root of this project we are making it easier on users who may want to use stack to build this project or like me need to try an alternative way to build it.

Reply:
I had a go at getting something like `stack install --resolver lts-13.9` going in one hit but haven't found something that works that way yet. Doing `stack init --resolver lts-13.9` on this repo is not an easy route to installation either whereas the `stack.yaml` of this pull request does allow a one line install with stack.

```
> stack init --resolver=lts-13.9
Looking for .cabal or package.yaml files to use to init the project.
Using cabal packages:
- ./
- fixtures/
- fixtures/doc/tutorial/
- fixtures/servant-client-core/
- fixtures/servant-client/
- fixtures/servant-docs/
- fixtures/servant-foreign/
- fixtures/servant-server/
- fixtures/servant/

Following packages have duplicate package names:
- haskell-ci.cabal
- fixtures/haskell-ci.cabal

Packages with duplicate names will be ignored.
Packages in upper level directories will be preferred.

Selected resolver: lts-13.9
Resolver 'lts-13.9' does not have a matching compiler to build some or all of your package(s).
ghc-8.6.3 cannot be used for these packages:
    - servant
    - servant-client
    - servant-client-core
    - servant-server
base version 4.12.0.0 found
    - servant requires >=4.7 && <4.11
    - servant-client requires >=4.7 && <4.11
    - servant-client-core requires >=4.7 && <4.11
    - servant-server requires >=4.7 && <4.11

This may be resolved by:
    - Using '--omit-packages to exclude mismatching package(s).
    - Using '--resolver' to specify a matching snapshot/resolver
```